packages/content/test/messages/Mail.test.ts

```diff
@@ -61,4 +83,24 @@ describe("Mail", function () {
         expect(error).toBeDefined();
         expect(error.message).toBe("Mail.to :: Value is not defined");
     });
+
+    test("should throw an Error if an invalid bodyFormat is passed", function () {
+        let error: any;
+
+        try {
+            Serializable.fromUnknown({
+                "@type": "Mail",
+                to: ["did:e:a-domain:dids:anidentity"],
+                cc: [],
+                subject: "aSubject",
+                body: "aBody",
+                bodyFormat: "InvalidFormat"
```

Comment:
Consistency with other input values.

```suggestion
                bodyFormat: "anInvalidFormat"
```